src/providers/formattingProvider.ts

```diff
@@ -235,11 +237,13 @@ export class FormatProvider implements vscode.DocumentFormattingEditProvider {
             }
         }
 
+        formattedDocument = removeEmptyLines(
```

Comment:
Let's add a formatting test to this one, esp with these values:

- -1
- 0
- 1
- 2
- 3

Reply:
Done in last commit. Please check it.